test/api.js

```diff
@@ -307,12 +307,23 @@ function generateTests(prefix, apiCreator) {
 			});
 	});
 
-	test(prefix + 'change process.cwd() to a test\'s directory', function (t) {
+	test(prefix + 'run from package.json folder by default', function (t) {
```

Comment:
I changed things to reflect what I suggest here. The commit history is a bit of a mess now though, due to the rebase from master happening before a pull/push. My bad ^^